devel/gitlint/Portfile

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                gitlint
+version             0.13.1
+categories-prepend  devel
+platforms           darwin
+license             MIT
+supported_archs     noarch
+
+python.default_version \
+                    39
```

Comment:
Is using python39 necessary? If this will work with python38, I think we should use that by default, to match [the default of the python-1.0 portgroup](https://github.com/macports/macports-ports/blob/master/_resources/port1.0/group/python-1.0.tcl#L76).

Reply:
Thanks for the suggestion @ryandesign. I've updated it to `python38` and added you as a co-author.